.github/workflows/go.yml

```diff
@@ -29,14 +29,15 @@ jobs:
       run: ./test
     - name: Run linter
       uses: golangci/golangci-lint-action@v2
+      if: ${{ matrix.go-version != '1.18.x' }}
```

Comment:
https://github.com/golangci/golangci-lint/issues/2649#issuecomment-1070528726 might be useful for getting lints working.